app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+const mongoose = require('mongoose');
+const url = 'mongodb+srv://Immanuel50:chukaglorY55@cluster0-ggn2t.mongodb.net/bucket-list-app?retryWrites=true&w=majority';
+mongoose.connect(url, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+});
+
+
+
+app.get('*', (req, res) => {
+    res.send('<h1>Page Not Found!</h1>')
```

Comment:
Instead of doing this, do this 

create an endpoint called `health` which is used to check the state of your app 

`app.get('/health', (req, res) => {
  res.status(201).send({
message: 'App started successfully'})
})`

`app.get("*", (req, res) => {
    res.status(201).send({
        message: "My Bucketlist Api"
    })
})`